src/net.h

```diff
@@ -199,7 +200,9 @@ class CNodeStats
     bool fInbound;
     int nStartingHeight;
     uint64_t nSendBytes;
+    mapCmdSize_t mapSendBytesPerCmd;
```

Comment:
Nit: don't use `_t` for the type name.
- I've learned recently (think I heard it from @gmaxwell) that this is reserved by the C standard for system types
- We don't use this convention for any of our own types